lib/dock.rb

```diff
@@ -0,0 +1,34 @@
+class Dock
+  attr_reader :name, :max_rental_time, :rental_time
+  def initialize(name, max_rental_time)
+    @name = name
+    @max_rental_time = max_rental_time
+    @revenue_multiplier = []
+    @rental_time
+  end
+
+  def rent(boat, renter)
+    @revenue_multiplier << boat.price_per_hour
+    @rental_time = 0
```

Comment:
This line means that _every_ time you call the `rent` method, you are resetting the `@rental_time` to zero - I don't think this is what you want.